src/scripts/main.js

```diff
@@ -1,3 +1,59 @@
 'use strict';
 
 // write code here
+const table = document.querySelector('.field');
+const appendRowButton = document.querySelector('.append-row');
+const removeRowButton = document.querySelector('.remove-row');
+const appendColumnButton = document.querySelector('.append-column');
+const removeColumnButton = document.querySelector('.remove-column');
+
+const MIN_COUNT = 2;
+const MAX_COUNT = 10;
+
+const updateButtonStates = () => {
+  const rowCount = table.rows.length;
+  const columnCount = table.rows[0].cells.length;
```

Comment:
Ensure that the table has at least one row and one cell initially, otherwise accessing `table.rows[0].cells.length` may throw an error if the table is empty.